### PR TITLE
Documentation for https://github.com/OctopusDeploy/Issues/issues/4738

### PR DESCRIPTION
**Pull request by johnsimons**

### CODE REVIEW

File: docs/deployment-examples/azure-deployments/running-azure-powershell/index.md [hunk 1]
```diff
@@ -28,7 +26,6 @@ Octopus Deploy provides a *Run an Azure PowerShell Script* step type, for exec
...
-:::
\ No newline at end of file
+### Staying up to date
+
+Octopus Deploy ships with a version of the Azure PowerShell Modules and Azure PowerShell CLI so you can deploy applications as soon as you install Octopus Deploy. Microsoft Azure is changing very quickly, introducing more application services and PowerShell commands frequently. You can use the built-in cmdlets or/and CLI if they cover everything you need, or you can configure Octopus Deploy to use your own [custom version of the Azure PowerShell cmdlets](configuring-the-version-of-the-azure-powershell-modules.md) or [custom version of the Azure PowerShell CLI](configuring-the-version-of-the-azure-powershell-cli.md).
```
Comment (benPearce1) [inline docs/deployment-examples/azure-deployments/running-azure-powershell/index.md]: I think it should be `and/or`

Comment (johnsimons) [reply docs/deployment-examples/azure-deployments/running-azure-powershell/index.md]: fixed

**Comment by matt-richardson** [comment]
Is this a good time to add something about how (some of) this functionality is not available in cloud? Or how cloud users can do it?

**Comment by johnsimons** [comment]
@matt-richardson I can add doco for it, do you know what features are not available ?

**Pull request merged**